src/Components/Comments.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleComments } from "../Utils/Api";
+import '../App.css';
+
+export const Comments = () => {
+
+    const { id } = useParams();
+  
+    const [comments, setComments] = useState([]);
+  
+
+    useEffect(() => {
+        getArticleComments(id).then((commentsFromApi) => {
+          setComments(commentsFromApi);
+        });
+      }, []);
```

Comment:
Could do with a loading pattern for while the comments are being fetched by the API